flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -1097,21 +1008,6 @@ public ArchivedExecutionConfig archive() {
 
     // ------------------------------ Utilities  ----------------------------------
 
-    public static class SerializableSerializer<T extends Serializer<?> & Serializable>
-            implements Serializable {
```

Comment:
Given that the SerializableSerializer is part of the public API, we cannot simply remove it. Additionally, we should avoid directly updating any return types of Public API that use SerializableSerializer. I suggest to maintain the current implementation without changes.

Reply:
OK, I'll revert the related changes, thanks for reminding